admin_guide/managing_networking.adoc

```diff
@@ -903,3 +903,50 @@ in all other projects. The project to be isolated must first be configured to
 opt in to isolation by setting the proper annotation on its Namespace object,
 and then creating `NetworkPolicy` objects indicating the incoming connections to
 be allowed.
+
+. Create the `NetworkPolicy` objects:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: allow-http-and-https
+spec:
+podSelector:
+ingress:
```

Comment:
`podSelector` and `ingress` should each be indented 2 spaces (and then everything under `ingress` also needs to be indented to match). So basically, everything after `spec` should be indented 2 spaces from where it is now